### DISCUSSION TITLE
In my tests, after use MINIO_COMPRESS I can't use this file in another min.io server

### DISCUSSION THREAD

**Discussion by StephanyBatista** [minio/minio#11486] [Q&A] [answered]
We are testing min.io to be a file server in our company. Using the environment MINIO_COMPRESS everything runs correctly. However, when I pick up the file compressed and I manually send (by server folder) it to another server min.io, this file does not open more.

My doubt: how does compress in min.io works? I saw that when I sent a file to min.io, the file gets compressed and when I get it, it is de-compressed, but If I have a problem with my container and I need to up another container, my pdf's stop opening?

**Comment by harshavardhana** [comment, marked as answer]
> We are testing min.io to be a file server in our company. Using the environment MINIO_COMPRESS everything runs correctly. However, when I pick up the file compressed and I manually send (by server folder) it to another server min.io, this file does not open more.
> 
> My doubt: how does compress in min.io works? I saw that when I sent a file to min.io, the file gets compressed and when I get it, it is de-compressed, but If I have a problem with my container and I need to up another container, my pdf's stop opening?

Once you compress, you can't read it again from the backend @StephanyBatista - you should access it through MinIO S3 API

**Reply by StephanyBatista** [reply]
Yes, I'm accessing from the MinIO S3 API, but my problem comes when I restore this file to another backend and try to access it by API.

**Reply by harshavardhana** [reply]
if you are restoring in FS mode, you need to make sure you also copy the relevant `fs.json` without this metadata, server doesn't know that object is compressed or not.

**Reply by StephanyBatista** [reply]
Ok, thanks.

I saw this file and I will try now
